src/Contexts/Adaptors/EVMAdaptors/helpers.ts

```diff
@@ -19,9 +19,9 @@ const getRpcProviderFromHttpUrl = (url: string) => {
       user: urlInstance.username,
       password: urlInstance.password,
     };
-    return new ethers.providers.JsonRpcProvider(urlInfo);
+    return new ethers.providers.StaticJsonRpcProvider(urlInfo);
```

Comment:
What is StaticJsonRpcProvider?

Reply:
`StaticJsonRpcProvider` doesn't make a call to get `chainId` per each request. So if we don't change the network then it should be fine.

Reply:
Here is also a [documentation](https://docs.ethers.io/v5/api/providers/jsonrpc-provider/#StaticJsonRpcProvider) for this top.